clinica/pydra/machine_learning_spatial_svm/pipeline.py

```diff
@@ -0,0 +1,131 @@
+import pydra
+from pydra import Workflow
+
+from clinica.pydra.engine import clinica_io
+
+
+def _build_query(parameters: dict) -> dict:
+    input_name = parameters["orig_input_data"].replace("-", "_")
+    if input_name == "pet_volume":
+        query = {}
+    elif input_name == "t1_volume":
+        query = {}
+    return query
```

Comment:
This function isn't doing anything atm. Could you move the query building logic in it ?

Reply:
Yes. I thought i would need to build a more complex query.